changelog_unreleased/api/15123.md

```diff
@@ -0,0 +1,3 @@
+#### fix: add missing `printer.preprocess` definition (#15123 by @so1ve)
+
+Adds missing types for `printer.preprocess`.
```

Comment:
We can just put type defination here. Something like:

````text
```diff
export interface Printer<T = any> {
+  preprocss
}
```
````